ussologin/ussologin.go

```diff
@@ -115,10 +117,13 @@ func LoginWithToken(client *http.Client, ussoAuthUrl string, tok *usso.SSOData)
 
 // TokenStore defines the interface for something that can store and returns oauth tokens.
 type TokenStore interface {
-	// Put stores an Ubuntu SSO OAuth token.
+	// Put stores an Ubuntu SSO OAuth token. The token should be
+	// retireved from the store using Get(tok.TokenName).
 	Put(tok *usso.SSOData) error
-	// Get returns an Ubuntu SSO OAuth token from store
-	Get() (*usso.SSOData, error)
+
+	// Get returns an Ubuntu SSO OAuth token, identified by name,
+	// from store
+	Get(name string) (*usso.SSOData, error)
```

Comment:
I'm not sure I'm keen on this. I think it works better as a straight store for a token with no regard to what that token is. The caller can check if the retrieved token matches their expectation if it likes.